toolchain/src/integration-test/resources/OnionArchitectureDemo/LICENSE

```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
Is adding a separate license file for the included demo project necessary?

Reply:
I do not know. The OnionArchitectureDemo project is from a separate repository (with the same authors). I think it is safer and "cleaner" to keep the license.

Reply:
I don't know exactly, either.
We already have another copy of the license in `owl-creator/LICENSE`.
Since it is exactly identical, probably one toplevel `LICENSE` file would be enough.
But we can also leave it as is.